storage/migrations/0009_oasis_3_emerald_init.up.sql

```diff
@@ -0,0 +1,90 @@
+-- Indexer state initialization for the Emerald ParaTime, after the Damask Upgrade.
```

Comment:
We are not able to generate this yet right, but I suppose this is a fixed thing.

Reply:
We aren't able to generate it, nor should we need to since runtimes don't have dump+restore upgrades, only state migrations.

I think there may be some utility in the future of writing a tool that takes in the modules a paratime supports and generates a file like this, though. But that probably doesn't save net time until we have a heck ton more runtimes to index